package.json

```diff
@@ -192,7 +192,8 @@
     }
   },
   "scripts": {
-    "devserver": "NODE_ENV=devserver nodemon .reaction/devserver/index.js",
+    "devserver": "npm run reaction-node",
+    "reaction-node": "NODE_ENV=reaction-node nodemon ./imports/node-app/index.js",
```

Comment:
Did we decide we want to keep this as `devserver` here?

Reply:
Yes, I'll adjust in the other branch.